code/__defines/items_clothing.dm

```diff
@@ -124,6 +124,7 @@
 #define LEG_LEFT    0x40
 #define LEG_RIGHT   0x80
 #define LEGS        0xC0    //  LEG_LEFT | LEG_RIGHT
+#define FEET        0x180  // FOOT_LEFT | FOOT_RIGHT
```

Comment:
???

Reply:
Whatever you trying to do it will not work.

Reply:
There is feet slot for shoes but why is there no bit flag for them ?

Reply:
Ther is no `FOOT_LEFT` or `FOOT_RIGHT` define. Same for `HAND_X` and `HANDS`.
So that defines have no support in code.
Check usage of that define and you will see.